ddmrp_production_equivalent/models/mrp_production.py

```diff
@@ -36,8 +36,12 @@ def _generate_raw_move(self, bom_line, line_data):
         product_equivalent = self._get_product_equivalent(bom_line,
                                                           line_data['qty'])
         if product_equivalent:
+            body = _('[%s] %s has been replaced by [%s] %s.' %
+                     sm.product_id.default_code, sm.product_id.name,
```

Comment:
What about use `name_get()[0][1]` instead of a manual `[%s] %s` in order to get the full name if there is an inherited method changing this one (as product variants case)?